spec/support/fake_heroku.rb

```diff
@@ -6,8 +6,9 @@ def initialize(args)
   end
 
   def run!
-    if @args.first == "plugins"
-      puts "heroku-pipelines@0.29.0"
+    if @args.first == "help"
+      # spaces in front are intentional
+      puts "  pipelines      #  manage collections of apps in pipelines"
```

Comment:
The real `heroku help` outputs two spaces in the beginning. In the grep regex the spaces are optional. So I can leave them out of the test. But than the criteria of the test not correct, is it? I can leave the comment out if it fits better in you coding style.